.github/workflows/publish-rust-crate.yml

```diff
@@ -92,8 +95,8 @@ jobs:
 
       - name: Set Git Author
         run: |
-          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions@github.com"
+          git config --global user.name "github-actions"
```

Comment:
Is this change needed?

Reply:
It should avoid seeing the `github-actions[bot]` as a contributor.

Reply:
Why do we want to avoid seeing `github-actions[bot]` as a contributor? I thought that was a good thing so that commits are associated with real github users :sweat_smile: